friends/src/addFriend.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+
+export default class Add extends React.Component {
+
+  constructor(props) {
+    super(props)
+    
+    this.state ={
+      name: '',
+      age: '',
+      email: '',
+      id: ''
```

Comment:
Instead of having this updated in the onChangeHandler, you can just put it here since you already know what you want it to be 
```suggestion
      id: this.props.friends.length + 1
```